tests/test_iclr_conference_v2.py

```diff
@@ -90,7 +92,9 @@ def test_create_conference(self, client, openreview_client, helpers, profile_man
 
         helpers.await_queue()
 
-        assert openreview_client.get_group('ICLR.cc/2024/Conference')
+        venue_group = openreview_client.get_group('ICLR.cc/2024/Conference')
+        assert venue_group
+        assert venue_group.content['date']['value'] == f'Abstract Registration: {abstract_date.strftime("%b %d %Y")} 12:00AM UTC-0, Submission Deadline: {due_date.strftime("%b %d %Y")} UTC-0'
```

Comment:
make sure you fix the test, it expects to have 12:00AM in the submission deadline too.

Reply:
Oh yes, just realized this. Fixed now.